static/js/narrow.js

```diff
@@ -431,6 +435,7 @@ exports.by_conversation_and_time = function (target_id, opts) {
 
     if (original.type !== 'stream') {
         args.push({operator: "pm-with", operand: original.reply_to});
+        args.push({operator: "huddle-with", operand: original.reply_to});
```

Comment:
Remove this change -- it is not correct.  This logic is for the fully specified narrow for a conversation, which is already the existing `pm-with` argument.